pos_hide_cost_price_and_margin/readme/DESCRIPTION.md

```diff
@@ -1,2 +1,2 @@
 This module extends the functionality of point of sale to hide cost and
-margin on product detail.
+margin on product detail. Besides odoo native's options, it supports to hide cost and margins among administrators when most of users are granted 'Manager' in POS
```

Comment:
I would suggest a more simple description:
- By default since odoo v16.0, Margin/Cost is hidden for PoS/User
- But in some contexts you also want it hidden for PoS/Manager
- Installing this module will hide it for every user